### ISSUE TITLE
Extra Utilities 2 conflits with Optifine since alpha 1.1.x

### ISSUE THREAD

**Issue by NiwatoriChan** [rwtema/extrautilities_old_issues#1039]
The game crashes.

**Comment by iKurama** [comment]
"I do not support Optifine. Optifine makes a large amount of changes to the way rendering works compared to Forge/Vanilla and I cannot garauntee that these changes will not break things. If you are using Optifnie and are having a problem (especially involving rendering) please check to see if it happens without Optifine enabled before reporting."

**Comment by NiwatoriChan** [comment]
Already done. The game starts without optifine or without extra utilities. The game performances are just so bad without optifine.

**Comment by DatrixTHLK** [comment]
Optifine Preview build C4 works. If you have bad framerate it is better to not get rid of Optifine.

**Comment by NiwatoriChan** [comment]
Thank you ! It worked.

**Comment by DatrixTHLK** [comment]
You're welcome. Since it worked, could you close the issue, please?

**Comment by KamranMackey** [comment]
@NiwatoriChan Maybe if your performance is bad, maybe consider spending money towards getting a better machine.

**Comment by DatrixTHLK** [comment]
@KamranMackey You assume he has "bad performance issues"? Buying a new better performance computer is much more expensive than you claim to realize, also, no one is made of money. Better performance doesn't always mean better. You will still have the same issues with the "better performance computer" that you did with the "bad performance computer". Sure you may get more mods to work but it will still cause issues with framerate no matter how many mods you put in. 

@NiwatoriChan Honestly, quality over quantity. Don't spend money you don't have to just to get mods to work. This is why us "bug testers" are here to give reports to mod authors to make sure they can make their mod optimized or have less issues (whether they want to or not is up to them). What is your Ram and Video Card of your computer btw?

IMHO stick with Optifine. ;)

**Comment by NiwatoriChan** [comment]
Thank you @DatrixTHLK and I think you're right, I'm playing triple A s without problems, so I won't spend more money to buy another computer. My two computers are not able to run Modded Minecraft(especially 1.10.2),  well without optifine. I have an AMD 1050 t 8gb with a Radeon 270x 4GB of VRAM and my laptop, a brand new inspiron 7559,  has an i7 6700hq 8gb ram with a NVIDIA 960m 4GB of VRAM.

@KamranMackey  all my friends has issues with framerate in 1.10.2. Pretty much all mods are in early release. I red an article from asie, the maintener of BuildCraft that's calling there's a lot of issue with MCP and Forge.

Furthermore, modded minecraft servers can run on cheap machines without an issue. In client level, oups. But we want better performance and  It's why we have mods like FastCraft or Optifine to speed up our games.

Cofh was attempting to make the dev in modded minecraft easier, there's no news anymore. If we need to change all our code in each version of minecraft, I can understand that is hard to optimise all the code each time. **We don't pay people doing code for mods**.

Personnly, I wanted to start modding for minecraft, but recoding my base code each time scares me...

Sorry for my not perfect  english btw, I'm not a native english speaker.

**Comment by KamranMackey** [comment]
@DatrixTHLK A mod called BetterFps exists that just improves framerate. Optifine is unneeded in these instances, also, Optifine breaks almost everything.

**Comment by DatrixTHLK** [comment]
o_O Well...this is the first time I have explained anything, well enough, to make someone delete their comment about buying a "better system". I am proud of myself.  :D  Of course now others will think I'm talking to a person who doesn't exist...

I am very verbose when it comes to knowing about a subject. Most people (uneducated ones or lazy) don't like this because they don't like to read. _rolls eyes_

@NiwatoriChan

> I have an AMD 1050 t 8gb with a Radeon 270x 4GB of VRAM and my laptop, a brand new inspiron 7559, has an i7 6700hq 8gb ram with a NVIDIA 960m 4GB of VRAM.

That is high-end stuff you have. You shouldn't have any issues with playing modded minecraft let alone framerate issues. The only reason I knew about issues with performance and a new more powerful system is because the last computer could only handle at most 85 mods with optifine (some glitching issues while streaming). The new computer can handle 160 mods with Optifine plus more if I so choose. My processor is AMD 8350 4.34Ghz with 2x4gb (8gb) of Ram (custom built PC desktop computer), so it should, in theory, be able to handle 160 mods in minecraft with ease. Sadly, without Optifine there are issues.

> Cofh was attempting to make the dev in modded minecraft easier, there's no news anymore. If we need to change all our code in each version of minecraft, I can understand that is hard to optimise all the code each time. 

First of all, I would like to say as much as Cofh has made great mods, they have not done well in updating any of them to the recent version of minecraft (along with however many other people mistakenly believe that 1.8+ is unmoddable). Many people have fallen for that little ploy of theirs in saying that 1.8+ has many changes and we don't like them hence we will stay at 1.7.10 and keep moding for that version (being the close minded people they are). My personal opinion on that is that 1.8 made it so much more smoother when attacking mobs and no aim-botting and less glitching and freezing issues. This is no more than a baseless charge, seeing as how mods are ported to the newest version of minecraft and still able to function. The modders who work on them now prove to those other modders 1.8+ is still moddable.

In 1.7.2/10, It was really frustrating when you went to attack mobs, they could somehow hit you before you hit them. The skeletons had aim-bots. With 1.8+ they aren't aimbotting. The combat update at first was a bit overwhelming seeing as I had to get used to something new. The new change is a very nice addition to minecraft. I'm used to it. I don't see the big deal everyone makes about new changes to minecraft.

A ton of mods do the same thing CoFH's mods do but better. EU for example in 1.7.2 had conveyor belts. This was also, something CoFH had in one of their mods where the floor would push the mobs to the end point in a mob farm. 

What I'm saying is that due to this, CoFH and their mods aren't well liked by me and I don't use them since Minecraft is at 1.10.2 now, (but that is my own choice).  I wouldn't hold my breath on any news from CoFH's Team seeing as there hasn't been any news in a while. If there is any news then I'd be surprised like a gift given without notice.  

Being a web developer, though, I know the issues of optimizing and yes, it is tough.

> We don't pay people doing code for mods.

This is true in a way, but they can get a donation or support through patreon (depending on who and if they want to).

**BOTTOM LINE:** Like everything else in life, take it a few steps at a time. Getting overwhelmed is only natural when facing a new challenge because it happens, even to me. I'd hate for mods to get left behind because someone burned themselves out on it. If you are going to start modding do it for the fun of it, treat it like you would any hobby. If people think you deserve money for what you do then that will be a bonus appreciation. Saying thank you always helps in the meantime just to let modders know they are, at least, in some way appreciated.

**Comment by asiekierka** [comment]
@NiwatoriChan What article are you referring to? I wrote an article that explained what goes into making an API like MCP/Forge, I don't think I wrote any actual critique of them.
